lib/cocoapods-pod-merge/Main.rb

```diff
@@ -334,7 +334,7 @@ def merge(merged_framework_name, group_contents, podfile_info)
 
       # Create the local podspec
       Pod::UI.puts "\tCreating Podspec for the merged framework".magenta
-      create_podspec(merged_framework_name, pods_to_merge, PodspecInfo.new(frameworks.uniq, prefix_header_contents.uniq, private_header_files.uniq, resources.uniq, script_phases.uniq, compiler_flags.uniq, libraries.uniq, prepare_command.uniq, resource_bundles, vendored_libraries.uniq, swift_version), mixed_language_group)
+      create_podspec(merged_framework_name, pods_to_merge, PodspecInfo.new(frameworks.uniq, prefix_header_contents.uniq, private_header_files.uniq, resources.uniq, script_phases.uniq, compiler_flags.uniq, libraries.uniq, prepare_command.uniq, resource_bundles, vendored_libraries.uniq, swift_version), mixed_language_group, podfile_info)
```

Comment:
Where is this`podfile_info` being created? 🤔

Reply:
Oh this was created in this line https://github.com/grab/cocoapods-pod-merge/blob/c9830f819441a40ec89aea97495f04f1f795549c/lib/cocoapods-pod-merge/Main.rb#L49 I only passed it to be able to extract the deployment target.